zgw/zaken-api/src/main/kotlin/com/ritense/zakenapi/domain/ZaakInformatieObject.kt

```diff
@@ -28,10 +28,15 @@ data class ZaakInformatieObject(
     val zaak: URI,
     val aardRelatieWeergave: String,
     val titel: String? = null,
+    val identificatie: String? = null,
+    val vertrouwelijkheidaanduiding: String? = null,
     val beschrijving: String? = null,
     @JsonFormat(shape = JsonFormat.Shape.STRING)
     val registratiedatum: LocalDateTime,
     @JsonFormat(shape = JsonFormat.Shape.STRING)
     val vernietigingsdatum: LocalDateTime? = null,
-    val status: URI? = null
+    val status: URI? = null,
+    val informatieobjecttype: String? = null,
+    val trefwoorden: String? = null,
+    val formaat: String? = null
```

Comment:
Weird, I don't see these fields in the API specs:
.../zaken/api/v1/schema/#tag/zaakinformatieobjecten

Reply:
I think these were added to the wrong file